### DISCUSSION TITLE
Will starstruck badge go away if people unstar your repo?

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [Schweinepriester/github-profile-achievements#175] [Q&A] [answered]
Will the starstruck badge go away if people unstar your repo?

**Comment by number0x0001** [comment]
I assume so

**Comment by number0x0001** [comment, marked as answer]
Try temporarily unstarring this repo and see what happens to the badge: https://github.com/number0x0001/Seanux-Linux-Files-Scripts @thenithinbalaji

**Reply by thenithinbalaji** [reply]
Your badge didnt go @number0x0001

**Reply by thenithinbalaji** [reply]
That's Great!

**Reply by number0x0001** [reply]
> Can you star my repo back https://github.com/thenithinbalaji/YES-or-NO

I starred and unstarred, so now you have the badge.

**Reply by thenithinbalaji** [reply]
The Starstruck badge won't go away even if people unstar the repo once it reaches 16 stars or any other badge earnable number of stars as demonstrated by number0x0001 

https://github.com/Schweinepriester/github-profile-achievements/discussions/175#discussioncomment-3158067

**Comment by thenithinbalaji** [comment]
The Starstruck badge won't go away even if people unstar the repo once it reaches 16 stars or any other badge earnable number of stars as demonstrated by @number0x0001 

https://github.com/Schweinepriester/github-profile-achievements/discussions/175#discussioncomment-3158067

**Comment by quartesa37** [comment]
No

**Reply by thenithinbalaji** [reply]
Why is no being marked as Spanish lmao

![image](https://user-images.githubusercontent.com/73932121/179285418-52964026-d2f0-4869-97a2-b31aba8b65b2.png)

**Reply by quartesa37** [reply]
I don't know

**Comment by quartesa37** [comment]
Yes> Will the starstruck badge go away if people unstar your repo?

**Reply by Crushoverride007** [reply]
yes it wont count, til you get the achievement, can you please leave one here https://github.com/Crushoverride007/Crushoverride007

**Comment by warriorwizard** [comment]
please leave a star here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
I'll star others,
btw thanks in advance

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts
